packages/yoroi-extension/app/styles/themes/revamp/dark-theme-base.js

```diff
@@ -0,0 +1,90 @@
+//@flow
+
+const cyan = { '400': '#59B1F4', '100': '#F2F9FF' };
+const yellow = { '500': '#ECBA09', '100': '#FDF7E2' };
+const magenta = {
+  main: '#FF1351',
+  '700': '#CF053A',
+  '600': '#E80742',
+  '500': '#FF1351',
+  '300': '#FBCBD7',
+  '100': '#FFF1F5',
+};
+
+export const darkTheme = {
+  name: 'revamp',
```

Comment:
I think the team name should be unique. 
```suggestion
  name: 'dark-revamp',
```

Reply:
I used the approach found in the docs, they don't specify the theme name but the mode. I guess we can remove that since is not used or just rename it like you did. (https://mui.com/material-ui/customization/dark-mode/#dark-mode-with-a-custom-palette)